InsightsFrontend/src/app/components/release-catalogus/release-catalogus.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, Input, OnDestroy, OnInit } from '@angular/core';
+import { Release } from '../../services/release.service';
+import { NgStyle } from '@angular/common';
+import { ModalComponent } from '../modal/modal.component';
+
+@Component({
+  selector: 'app-release-catalogus',
+  standalone: true,
+  templateUrl: './release-catalogus.component.html',
+  styleUrl: './release-catalogus.component.scss',
+  imports: [NgStyle, ModalComponent],
+})
+export class ReleaseCatalogusComponent implements OnInit, OnDestroy {
+  @Input() selectedRelease: Release | null = null;
+  modalOpen = false;
+  isSmallScreen = false;
+  private mediaQueryList: MediaQueryList | null = null;
+  private mediaListener: (() => void) | null = null;
+
+  toggleModal(): void {
+    this.modalOpen = !this.modalOpen;
+  }
+
+  ngOnInit(): void {
+    this.mediaQueryList = globalThis.matchMedia('(max-width: 540px)');
+    this.isSmallScreen = this.mediaQueryList.matches;
+    this.mediaListener = (): void => {
+      this.isSmallScreen = this.mediaQueryList!.matches;
```

Comment:
Isn't it better to use optional chaining instead of non-null assertion? That it would just return false.